src/sentry/db/models/fields/encrypted.py

```diff
@@ -56,7 +56,24 @@ def get_db_prep_value(self, value, *args, **kwargs):
     def to_python(self, value):
         if value is not None and isinstance(value, six.string_types):
             value = decrypt(value)
-        return super(EncryptedPickledObjectField, self).to_python(value)
+
+        # The following below is a copypaste of PickledObjectField.to_python of
+        # v1.0.0 with one change: We re-raise any baseexceptions such as
+        # signals. 1.0.0 has a bare `except:` which causes issues.
+
+        if value is not None:
```

Comment:
Quite honestly I would always reraise. If depickling fails, why would I ever want the raw value to be returned?

Reply:
i thought this method had to be idempotent because of django?

Reply:
Apparently this is expected to be idempotent. Let's try it like this first.